website/views/pages/device-management.ejs

```diff
@@ -44,57 +44,47 @@
         <% } %>
 
         </div>
-        <div purpose="button-row" class="d-flex align-items-center pt-3">
-          <a purpose="cta-button" href="/contact">Talk to an engineer</a>
-          <animated-arrow-button  href="/register">Try it yourself</animated-arrow-button>
-        </div>
+        <div purpose="button-row" class="d-flex justify-content-start align-items-center">
+              <a purpose="cta-button" href="/contact">Talk to an engineer</a>
+              <a purpose="video-button"  @click="clickOpenVideoModal('fleet-in-three-minutes')"><img alt="Play" class="d-inline" src="/images/icon-play-video-32x32@2x.png"> Watch demo <span>3 mins</span></a>
+            </div>
       </div>
       <logo-carousel></logo-carousel>
 
-    <%/* Debunk the cross-platform myth */%>
-      <div purpose="page-section" class="pb-0">
-        <div purpose="feature" class="d-flex flex-md-row flex-column-reverse justify-content-between mx-auto align-items-center">
-          <div purpose="feature-text" class="d-flex flex-column">
-            <h2>Debunk the cross-platform myth</h2>
-            <p>Conventional wisdom holds that cross-platform MDM solutions don't hold up at scale. Few, if any, have lived up to the hype.</p>
-            <p>Instead, Fleet lets you work directly with data and events from each native operating system. So while you can still use familiar concepts like smart groups and extension attributes, you don’t get stuck “talking Windows” to Apple devices.</p>
-          </div>
-          <div purpose="feature-image" class="right">
-            <img alt="Transparency for end users" src="/images/mdm-debunk-cross-platform-myth-528x377@2x.png">
-          </div>
-        </div>
+      <div purpose="page-section" class="pb-0 mt-5">
 
         <div purpose="three-column-features" class="mx-auto">
           <div purpose="responsive-feature-row" class="row d-flex flex-sm-row flex-column justify-content-lg-between justify-content-center align-items-center align-items-sm-start">
             <div purpose="feature-item" class="pl-0 col-12 col-sm-6 col-lg-4">
               <h5>Operating systems</h5>
-              <p>Enforce OS updates with Declarative Device Management (DDM), <a href="https://github.com/macadmins/nudge" target="_blank">Nudge</a>, and Windows Update from one console.</p>
+              <p>Manage Apple, Windows, Linux, and Android devices in one place. Fleet supports diverse, mixed environments with no vendor lock-in.</p>
             </div>
 
             <div purpose="feature-item" class="pr-0 pr-lg-4 col-12 col-sm-6 col-lg-4" >
               <h5>Automated enrollment</h5>
-              <p>Drop-ship devices to your end users with Apple Business Manager or Autopilot and let them set up their own account. No IT help needed.</p>
```

Comment:
Reason for change:

Allen: I'd prefer it to say "set up their own device" not "account."